xir/__init__.py

```diff
@@ -1,13 +1,40 @@
+from lark import lark
+
 from ._version import __version__
 from .decimal_complex import DecimalComplex
-from .parser import Transformer, parser
-from .program import Declaration, ObservableStmt, Program, Statement
+from .parser import Transformer, read_lark_file
+from .program import Declaration, ObservableStmt, Program, Statement, ObservableFactor
 from .validator import Validator
 
-__all__ = ["DecimalComplex", "Declaration", "ObservableStmt", "Program", "Statement", "Transformer"]
+__all__ = [
+    "DecimalComplex",
+    "Declaration",
+    "ObservableFactor",
+    "ObservableStmt",
+    "Program",
+    "Statement",
+    "Transformer",
+]
 
 
-def parse_script(script: str, **kwargs) -> Program:
+def parse_script(script: str, debug=True, **kwargs) -> Program:
```

Comment:
We should add a type hint for the debug flag and describe its effect in the docstring.
```suggestion
def parse_script(script: str, debug: bool =True, **kwargs) -> Program:
```